samples/application_development/code_relocation_nocopy/README.rst

```diff
@@ -51,3 +51,29 @@ Execution output
   Address of function_in_sram 0x20000001
   Address of var_sram_data 0x200000a4 (10)
   Hello World! nrf5340dk/nrf5340/cpuapp
+
+
+STM32 b_u585i_iot02a Discovery kit instructions
+***********************************************
+
+The b_u585i_iot02a has 64MB of external flash attached via OSPI. It is mapped
+to 0x70000000.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/application_development/code_relocation_nocopy
+   :board: b_u585i_iot02a
+   :goals: build flash
+   :compact:
```

Comment:
Move this section before `Execution output`

Reply:
section moved